src/_guides/language/language-tour.md

```diff
@@ -4547,8 +4547,10 @@ shared-memory threads running concurrently. However, shared-state
 concurrency is error prone and can lead to complicated code.
 
 Instead of threads, all Dart code runs inside of *isolates*. 
-Each Dart isolate has a single thread of execution and
-shares no mutable objects with other isolates.  
+Each Dart isolate uses a single thread of execution and
+shares no mutable objects with other isolates.
+Spinning up multiple isolates creates multiple threads of execution.
+This enables multi-threading without its primary drawback, race conditions.
```

Comment:
Should we link `race conditions` to the wikipedia page? I find their example pretty easy to read.